grpc-gateway/pb/clientConfiguration.proto

```diff
@@ -16,4 +16,8 @@ message ClientConfigurationResponse {
     string cloud_url  = 6;
     string cloud_authorization_provider = 7;
     string cloud_certificate_authorities = 8;
+    // exposes default command time to live in nanoseconds for CreateResource, RetrieveResource, UpdateResource, DeleteResource, and UpdateDeviceMetadata commands when it is zero value in the request. 0 - means forever.
+    int64 default_command_time_to_live = 9;
```

Comment:
in proto you have time_to_live and in deployment documentation TTL.

Reply:
for `default_command_ttl` you will see `DefaultCommandTtl` so I decided to use the full name of the parameter.

Reply:
But why you didn't unify it? Somewhere is TTL and somewhere else TimeToLive.

Reply:
Done -> `TTL` was replaced by `TimeToLive`.